modules/aws-bastion-instance/example/main.tf

```diff
@@ -48,8 +48,16 @@ EOF
   vpc_id               = var.vpc_id
 
   # Access
-  ssh_key_name        = "example-key"
-  allowed_cidr_blocks = [
+  ssh_key_name           = "example-key"
+  allowed_cidr_blocks    = [
     "18.202.145.21/32", # Boldare VPN IP
   ]
+  vpc_public_subnet_tags = {
+    "MyVpcSubnetType" = "Public"
+  }
+}
+
+resource "aws_iam_role_policy_attachment" "bastion_ssh_policy" {
```

Comment:
[name](https://www.terraform.io/docs/providers/aws/r/iam_policy_attachment.html#name) prop is required in `aws_iam_role_policy_attachment`